lnbits/extensions/boltz/templates/boltz/_api_docs.html

```diff
@@ -6,30 +6,88 @@
       style="padding: 5px 9px"
       alt=""
     />
+    <p><b>NON CUSTODIAL atomic swap service</b></p>
     <h5 class="text-subtitle1 q-my-none">
-      Boltz.exchange: Do onchain to offchain and vice-versa swaps
+      Providing trustless and account-free swap services since 2018. Move IN and
+      OUT of the lightning network and remain in control of your bitcoin, at all
+      time.
     </h5>
     <p>
-      Submarine and Reverse Submarine Swaps on LNbits via boltz.exchange API<br />
-    </p>
-    <p>
-      Link :
+      Link:
       <a target="_blank" href="https://boltz.exchange"
         >https://boltz.exchange
       </a>
-    </p>
-    <p>
+      <br />
+      README:
       <a
         target="_blank"
         href="https://github.com/lnbits/lnbits-legend/tree/main/lnbits/extensions/boltz"
-        >More details</a
+        >read more</a
       >
     </p>
     <p>
       <small
-        >Created by,
+        >Extension created by,
         <a target="_blank" href="https://github.com/dni">dni</a></small
       >
     </p>
   </q-card-section>
 </q-card>
+<q-card>
+  <q-card-section>
+    <h3 class="text-subtitle1 q-my-none">
+      <b>Fee Information</b>
+    </h3>
+    <span>
+      Every swap consists of 2 onchain transactions, lockup and redeem / refund,
```

Comment:
We don't call the second transaction "redeem" but "claim" transaction, but whatever you prefer as long as you are consistent about it

Reply:
changed to claim